website/static/js/search.js

```diff
@@ -46,8 +46,6 @@ var User = function(result){
     self.degree = result.degree;
     self.school = result.school;
     self.url = result.url;
-    self.wikiUrl = result.url+'wiki/';
```

Comment:
These variable are created under user model, but user don't have wikiUrl and fielUrl.